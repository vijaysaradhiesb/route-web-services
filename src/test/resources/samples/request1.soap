<soapenv:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:ws="http://ws.integclaims.itf.com">
    <soapenv:Header/>
    <soapenv:Body>
        <ws:FindRefData soapenv:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/">
            <username xsi:type="xsd:string">username</username>
            <password xsi:type="xsd:string">password</password>
            <TABLE_NAME xsi:type="xsd:string">ref_data</TABLE_NAME>
            <PARENT_CODE xsi:type="xsd:string">PURPOSE_OF_JOURNEY</PARENT_CODE>
            <COMPANY_CODE xsi:type="xsd:string">NOIDEA</COMPANY_CODE>
            <FILTER_CODE xsi:type="xsd:string">RESPOND</FILTER_CODE>
            <addShowAllEntry xsi:type="xsd:boolean">false</addShowAllEntry>
            <showAllCompanyCodes xsi:type="xsd:boolean">false</showAllCompanyCodes>
            <exludeEmptyCompanyCodes xsi:type="xsd:boolean">false</exludeEmptyCompanyCodes>
            <exludeEmptyFilterCodes xsi:type="xsd:boolean">false</exludeEmptyFilterCodes>
            <enforceParentRlship xsi:type="xsd:boolean">false</enforceParentRlship>
            <extraCodeToAdd xsi:type="xsd:string">SDP</extraCodeToAdd>
        </ws:FindRefData>
    </soapenv:Body>
</soapenv:Envelope>
