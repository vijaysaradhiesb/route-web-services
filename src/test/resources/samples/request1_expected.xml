<?xml version="1.0" encoding="UTF-8" standalone="yes"?><ns2:GenericWebServiceMessage xmlns="http://ws.integclaims.itf.com" xmlns:ns2="http://generic.message.integclaims.integ.com"><ns2:name>FindRefData</ns2:name><ns2:keyValuePair><ns2:key>username</ns2:key><ns2:valueType>STRING</ns2:valueType><ns2:stringValue>username</ns2:stringValue></ns2:keyValuePair><ns2:keyValuePair><ns2:key>password</ns2:key><ns2:valueType>STRING</ns2:valueType><ns2:stringValue>password</ns2:stringValue></ns2:keyValuePair><ns2:keyValuePair><ns2:key>TABLE_NAME</ns2:key><ns2:valueType>STRING</ns2:valueType><ns2:stringValue>ref_data</ns2:stringValue></ns2:keyValuePair><ns2:keyValuePair><ns2:key>PARENT_CODE</ns2:key><ns2:valueType>STRING</ns2:valueType><ns2:stringValue>PURPOSE_OF_JOURNEY</ns2:stringValue></ns2:keyValuePair><ns2:keyValuePair><ns2:key>COMPANY_CODE</ns2:key><ns2:valueType>STRING</ns2:valueType><ns2:stringValue>NOIDEA</ns2:stringValue></ns2:keyValuePair><ns2:keyValuePair><ns2:key>FILTER_CODE</ns2:key><ns2:valueType>STRING</ns2:valueType><ns2:stringValue>RESPOND</ns2:stringValue></ns2:keyValuePair><ns2:keyValuePair><ns2:key>addShowAllEntry</ns2:key><ns2:valueType>BOOLEAN</ns2:valueType><ns2:booleanValue>false</ns2:booleanValue></ns2:keyValuePair><ns2:keyValuePair><ns2:key>showAllCompanyCodes</ns2:key><ns2:valueType>BOOLEAN</ns2:valueType><ns2:booleanValue>false</ns2:booleanValue></ns2:keyValuePair><ns2:keyValuePair><ns2:key>exludeEmptyCompanyCodes</ns2:key><ns2:valueType>BOOLEAN</ns2:valueType><ns2:booleanValue>false</ns2:booleanValue></ns2:keyValuePair><ns2:keyValuePair><ns2:key>exludeEmptyFilterCodes</ns2:key><ns2:valueType>BOOLEAN</ns2:valueType><ns2:booleanValue>false</ns2:booleanValue></ns2:keyValuePair><ns2:keyValuePair><ns2:key>enforceParentRlship</ns2:key><ns2:valueType>BOOLEAN</ns2:valueType><ns2:booleanValue>false</ns2:booleanValue></ns2:keyValuePair><ns2:keyValuePair><ns2:key>extraCodeToAdd</ns2:key><ns2:valueType>STRING</ns2:valueType><ns2:stringValue>SDP</ns2:stringValue></ns2:keyValuePair></ns2:GenericWebServiceMessage>